sdk/src/main/java/com/horizen/transaction/exception/TransactionSemanticValidityException.java

```diff
@@ -0,0 +1,25 @@
+package com.horizen.transaction.exception;
+
+public class TransactionSemanticValidityException extends RuntimeException {
```

Comment:
Consider to inherit  TransactionSemanticValidityException from Exception because invalid transaction is something we are expecting and we always have to check it.